.travis.yml

```diff
@@ -2,7 +2,7 @@ language: node_js
 before_install:
   - npm install -g npm@latest
 node_js:
-  - 4
```

Comment:
Why remove Node v4? As long as it's LTS, I think we should still test against it.